src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
```

Comment:
```suggestion
    <main class="calendar calendar--month-length-31 calendar--start-day-sun">
```
Consider adding modifiers here to handle its change